pxtlib/docsrender.ts

```diff
@@ -230,6 +230,15 @@ namespace pxt.docs {
             return injectHtml(templ, mparams, ["ITEMS"])
         }
 
+        let availableLocales = theme.selectLanguage && theme.availableLocales && theme.availableLocales.length ?
+        theme.availableLocales : ["en"]
+
+        params["selectLang"] = lf("Select Language");
+        params["crowdin"] = theme.crowdinProject ?
```

Comment:
we should not send people direclty to crowdin. Instead send them to ``/translate``.